app/assets/stylesheets/new_styles.scss

```diff
@@ -1073,3 +1073,37 @@ span.disabled {
 .citation-box {
   overflow-x: auto;
 }
+
+a.orcid-link {
+  background-image: url(image-path('logos/orcid_16x16.png'));
```

Comment:
would be good if this image path was taken from the image dictionary. But I see there other hard-coded cases in this file so can update after merging